assets/js/types/type-guards/index.ts

```diff
@@ -13,7 +13,11 @@ export const isString = < U >( term: string | U ): term is string => {
 export const isObject = < T extends Record< string, unknown >, U >(
 	term: T | U
 ): term is NonNullable< T > => {
-	return ! isNull( term ) && typeof term === 'object';
+	return (
```

Comment:
I've updated this as it would have previously returned `true` if given a value of `[]`.